params/hope_buzzard.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 5
+    
+# e.g. "anas_krystallinus"    
+species.name <- "hope_buzzards"
```

Comment:
Thanks @Embuali1 could you change the species to something that @bobturneruk hasn't used please.

It can be anything you want as long as its unique.

Thanks @ns-rse